### PR TITLE
workflow: cucushift-installer-rehearse-aws-ipi-kubevirt-hypershift

### PR DESCRIPTION
**Pull request by LiangquanLi930**
https://issues.redhat.com/browse/OCPQE-12473
https://issues.redhat.com/browse/OCPQE-13742

add one hypershift pipeline for kubevrit platform for 4.12

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-kubevirt-hypershift-p2-f21

**Comment by LiangquanLi930** [comment]
https://issues.redhat.com/browse/OCPQE-12473

**Comment by LiangquanLi930** [comment]
@jianlinliu @liangxia @kuiwang02 Could you help to review this PR when you have time? thanks

File: ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/kubevirt/cucushift-installer-rehearse-aws-ipi-kubevirt-workflow.metadata.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+	"path": "cucushift/installer/rehearse/aws/ipi/kubevirt/cucushift-installer-rehearse-aws-ipi-kubevirt-workflow.yaml",
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/kubevirt/cucushift-installer-rehearse-aws-ipi-kubevirt-workflow.metadata.json:2]: Per https://github.com/openshift/release/pull/35798/files#r1091362272, it is calling `ovn` conf step explicitly, so suggest to move it to `cucushift/installer/rehearse/aws/ipi/ovn` folder.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 2]
```diff
@@ -478,6 +478,21 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-ingress-lbtype-nlb
+- as: aws-ipi-kubevirt-hypershift-p3-f21
```
Comment (kuiwang02) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:481]: @LiangquanLi930 following previous name rule, it should be `aws-ipi-kubevirt-hypershift-mgmt-p3-f21`, like `aws-ipi-ovn-hypershift-mgmt-p3-f21`

**Comment by LiangquanLi930** [comment]
@jianlinliu @kuiwang02 thanks for your comments, could you help to review again? thanks

**Comment by kuiwang02** [comment]
iamok from golang's view.

**Comment by LiangquanLi930** [comment]
@liangxia could you help to review this PR? thanks

**Pull request merged**